Author: aszlig <aszlig@nix.build>
Date:   Fri Mar 30 04:35:48 2018 +0200

    Remove SESSION_SECRET* from example config
    
    We're going to provide these secrets via environment variables and
    they're generated during first start of the server.
    
    Signed-off-by: aszlig <aszlig@nix.build>
    Filename: remove-session-secret-from-config.patch

diff --git a/config.json.example b/config.json.example
index 06b9dcdae..620ac3405 100644
--- a/config.json.example
+++ b/config.json.example
@@ -11,9 +11,6 @@
     "CRON_SAFE_MODE":"false",
     "CRON_SEMI_SAFE_MODE":"false",
     "MAINTENANCE_MODE": "false",
-    "SESSION_SECRET":"YOUR SECRET HERE",
-    "SESSION_SECRET_KEY": "1234567891234567891234567891234567891234567891234567891234567891",
-    "SESSION_SECRET_IV": "12345678912345678912345678912345",
     "ADMIN_EMAIL": "you@example.com",
     "MAILER_SOCKET": "/run/habitica/mailer.sock"
 }
